app/assets/javascripts/toc.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+(function(window, document) {
+  const initCurrentLinkListener = () => {
+    const scrollPadding = 125;
+    const currentClassName = 'Toc__link--current';
+    const tocLinkNodes = document.querySelectorAll('.Toc__link');
+    const headingNodes = document.querySelectorAll('h2.Docs__heading');
+
+    const getDistanceFromTop = (element) => {
+      if (!element.offsetParent) {
+        return element.offsetTop;
+      }
+      return element.offsetTop + getDistanceFromTop(element.offsetParent);
+    };
+
+    const getHeadingPos = (heading) => getDistanceFromTop(heading);
+
+    const setCurrentLink = (hash) => {
```

Comment:
Should we consider adding an [`aria-current`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current) attribute when highlighting the link?

Reply:
TIL `aria-current`. Great idea. I've added [this](https://github.com/buildkite/docs/pull/1851/commits/6c8639a7376e49bbe8968910889aab32b5f6cfa4#diff-2244aad9e766cfd2cf16ab727def0e5d7b36db14310f993e8f2ebf63d2a9b422R22)